plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -592,8 +621,11 @@ def _parse_get_trace_props(
         trace: BaseTraceType,
         hf_x: Iterable = None,
         hf_y: Iterable = None,
+        # TODO -> are these type hints correct
```

Comment:
Not really sure tbh..

Reply:
Me neither, will just remove this comment; as this is a hidden method.